.github/workflows/test.yml

```diff
@@ -208,6 +208,7 @@ jobs:
         pushd dist/images
           sudo cp -f ../../go-controller/_output/go/bin/ovn* .
           sudo cp -f ../../go-controller/_output/go/bin/hybrid-overlay-node .
+          make fedora-dev
```

Comment:
sudo?

Reply:
Added sudo, let's see if that makes CI happy.